### ISSUE TITLE
Doesn't block CoinNebula

### ISSUE THREAD

**Issue by R4z0rX** [xd4rker/MinerBlock#20]
Hi!
Today I noticed that powvideo.net was using a lot of my CPU and I find out that is using this:
var miner = new CoinNebula.Instance('powvideo.net', {throttle: 0.7}); if(!miner.isMobile()) miner.start();

Is there a way to block this too?

Thanks in advance! :)

P.D.: Btw, I'm using the last version of MinerBlock (1.1.15) in Chrome.

**Comment by xd4rker** [comment]
Thanks for reporting this. You can add these to the custom filters in the settings page :
```
*://*.1q2w3.website/*
wss://*.1q2w3.website/*
```
I will release an update soon, thanks!

**Comment by R4z0rX** [comment]
Awesome! Thanks for your reply.
Cheers! :)

**Comment by R4z0rX** [comment]
Hey, hi again!
And again Powvideo and CoinNebula.
This time I added this filters and it seems working:
*://*.1q2w3.life/*
wss://*.1q2w3.life/*
But still, it keep querying and querying to wss://ws02.1q2w3.life/proxy and wss://ws03.1q2w3.life/proxy and still use some % of the CPU, and I mean more than the rest of the opened pages, not at a constant rate but with peaks, like every second or so (I see this in the Chrome Task Manager).
Is there  way to block this?
Thanks in advance! :)

Visit this link to test it:
https://povwideo.cc/embed-97yllc5miqtc-920x360.html

**Comment by xd4rker** [comment]
Try these filters:

```
*://*.povwideo.cc/player7/jwpsrva.js*
wss://*.1q2w3.life/*
```

I'll release an update tonight, thanks!

**Comment by xd4rker** [comment]
Fixed in ba246863851986200be1d5a27871db076a7a3341.
